src/countItems.js

```diff
@@ -11,12 +11,20 @@
  * @returns {object} An object containing the count of each
  *  string in the input array
  */
-function countItems(array) {
-  // This is your job. :)
+function countItems(array,value) {
+  let count = 0;
+  for (let element of array) {
```

Comment:
You're using `of` here, which is fine for what you're trying to do, but it's an orange flag for me:

1. Are you using `of` here because you're copying it from some other place and using `in` elsewhere for some other reason?
2. Are you just not paying attention to the difference? Or don't think the difference will stand out to me?
3. Do you not really get what `of` does, so you don't see how you could use it where you use `in`?